__tests__/articles.test.js

```diff
@@ -0,0 +1,59 @@
+const app = require("../api/app");
+const request = require("supertest");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("/api/articles/:article_id", () => {
+  it("GET:200 responds with correct article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.article_id).toBe(1);
+        expect(typeof body.article).toBe("object");
+      });
+  });
+
+  it("GET:200 response object has the following properties: author, title, article_id, body, topic, created_at, votes, article_img_url", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty("author");
+        expect(body.article).toHaveProperty("title");
+        expect(body.article).toHaveProperty("article_id");
+        expect(body.article).toHaveProperty("body");
+        expect(body.article).toHaveProperty("topic");
+        expect(body.article).toHaveProperty("created_at");
+        expect(body.article).toHaveProperty("votes");
+        expect(body.article).toHaveProperty("article_img_url");
```

Comment:
You'd ideally want to check these are the correct types as well. 

Crutially, we'd want to make sure we've got back article_id of 1, as thats the one you've requested. At the moment, this matches for all